tests/baselines/reference/declarationEmitAliasExportStar.js

```diff
@@ -0,0 +1,44 @@
+//// [tests/cases/compiler/declarationEmitAliasExportStar.ts] ////
+
+//// [thingA.ts]
+export interface ThingA { }

```

Comment:
I can reproduce the error without `ThingA`:
**a.ts**
```ts
export interface I { }
```
**b.ts**
```ts
export * from "./a";
```
**c.ts**
```ts
import * as b from "./b";
export const f = (_: b.I) => {};
```

Reply:
Yep. I included ThingA in my issue report only to illustrate the difference between #19852 and #20657. No need to include it in the test.